transit/tests/api_test/mixin/user_mixin.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import (
+    User,
+    Permission
+)
+from django.contrib.contenttypes.models import ContentType
+
+from transit.models import UserManager
+
+
+class UserMixin:
```

Comment:
It's nice that you added this kind of helper early, I wouldn't go for `Mixin` in case of providing easy way to handle users in tests, rather use class attribute to which `TestUserHelper` is assigned. `Mixin` should be focused on something that class expects, e.g. implementation of abstracts. 
`_TEST_USERNAME`, `_TEST_PASSWORD` and additional information could be passed through  initializer with default values, like
```python
def __init__(self, username=_TEST_USERNAME, password=_TEST_PASSWORD, force_recreate=False, **user_data)
    existing_user = User.objects.filter...
    if existing_user and not force_recreate: 
        return existing_user 
    ...
```